File: Bot.js
const { Telegraf } = require('telegraf');

const bot = new Telegraf('YOUR_BOT_TOKEN');

bot.command('start', (ctx) => {
  ctx.reply('Готовлю колы!');
});

bot.on('text', (ctx) => {
  try {
    const messageText = ctx.message.text;

    if (messageText === 'На хуй' || messageText === 'Посадить на хуй') {
      ctx.reply('Себя нахуй посади');
    }

    if (messageText === 'Посадить на кол' || messageText === 'На кол') {
      const replyUserId = ctx.message.reply_to_message.from.id;
      const replyUserName = ctx.message.reply_to_message.from.username;
      const currentUserUsername = ctx.message.from.username;

      if (replyUserName === 'Indra_0000') {
        ctx.reply('Индру на кол нельзя!!!');
      } else if (replyUserName === 'zarinsiki') {
        ctx.reply('Любимую Каночку нельзя!!!!');
      } else {
        ctx.reply(`${replyUserName} был посажен на кол ${currentUserUsername}`);
      }
    }
  } catch (error) {
    ctx.reply('Ответь на сообщение, гений');
  }
});

bot.launch();

File: Bot.py
import telebot

bot = telebot.TeleBot("6310861018:AAEsFQjGMhAJH1OnQum-AO1JwKi_5SIbQ9w")

@bot.message_handler(commands=["start"])
def start_message(message):
    bot.send_message(message.chat.id, "Готовлю колы!")

@bot.message_handler(content_types=["text"])
def echo_message(message):
    try:
        if message.text == 'На хуй' or message.text.lower() == 'Посадить на хуй':
            bot.send_message(message.chat.id, 'Себя нахуй посади')
        if message.text == "Посадить на кол" or message.text.lower() == "На кол":
            reply_user_id = message.reply_to_message.from_user.id
            reply_user_name = message.reply_to_message.from_user.username

            if reply_user_name == "Indra_0000":
                bot.send_message(message.chat.id, 'Индру на кол нельзя!!!')
            elif reply_user_name == "zarinsiki":
                bot.send_message(message.chat.id, "Любимую Каночку нельзя!!!!")

            elif reply_user_name == None:
                bot.send_message(message.chat.id, f"@{message.reply_to_message.from_user.first_name} был посажен на кол @{message.from_user.username}")
            else:
                bot.send_message(message.chat.id, f"@{reply_user_name} был посажен на кол @{message.from_user.username}")
    except:
        bot.send_message(message.chat.id, "Ответь на сооб, гений")


bot.polling()
